### ISSUE TITLE
Language update throws PHP Strict standards

### ISSUE THREAD

**Issue by swissspidy** [wp-cli/wp-cli#3424]
This is a followup to #2915 / #2926. I'm still experiencing the issue using WP-CLI 0.24.1.

```bash
Strict Standards: Only variables should be passed by reference in phar:///usr/local/bin/wp/php/WP_CLI/CommandWithTranslation.php on line 169
```

**Comment by danielbachhuber** [comment]
> I'm still experiencing the issue using WP-CLI 0.24.1.

This is already fixed for 0.25.0 in #3380

**Comment by swissspidy** [comment]
Sorry & thanks then!
